Docs builds failing

I think I lack experience on how to solve this the best way. @minrk the following lines [breaks the docs build](https://readthedocs.org/projects/jupyterhub-kubespawner/builds/7758937/):

https://github.com/jupyterhub/kubespawner/blob/ed75fbc95c2674aa15f604da973de7cb60c85ba8/docs/source/conf.py#L78-L80

# Reproduction

```sh
$ cd html
$ conda env create environment.yml --name kubespawner-docs
$ source activate kubespawner-docs
$ make html
Running Sphinx v1.7.9

Configuration error:
There is a programable error in your configuration file:

Traceback (most recent call last):
  File "/home/erik/anaconda3/envs/kubespawner-docs/lib/python3.5/site-packages/sphinx/config.py", line 161, in __init__
    execfile_(filename, config)
  File "/home/erik/anaconda3/envs/kubespawner-docs/lib/python3.5/site-packages/sphinx/util/pycompat.py", line 150, in execfile_
    exec_(code, _globals)
  File "conf.py", line 80, in <module>
    release = pkg_resources.get_distribution("jupyterhub-kubespawner").version
  File "/home/erik/anaconda3/envs/kubespawner-docs/lib/python3.5/site-packages/pkg_resources/__init__.py", line 476, in get_distribution
    dist = get_provider(dist)
  File "/home/erik/anaconda3/envs/kubespawner-docs/lib/python3.5/site-packages/pkg_resources/__init__.py", line 352, in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
  File "/home/erik/anaconda3/envs/kubespawner-docs/lib/python3.5/site-packages/pkg_resources/__init__.py", line 895, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/home/erik/anaconda3/envs/kubespawner-docs/lib/python3.5/site-packages/pkg_resources/__init__.py", line 781, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'jupyterhub-kubespawner' distribution was not found and is required by the application

Makefile:20: recipe for target 'html' failed
make: *** [html] Error 2
```

--- a/docs/environment.yml
+++ b/docs/environment.yml
@@ -10,3 +10,4 @@ dependencies:
     - recommonmark==0.4.0
     - pyyaml
     - sphinx-copybutton
+    - setuptools

--- a/docs/requirements.txt
+++ b/docs/requirements.txt
@@ -1,0 +1,7 @@
+sphinx>=1.7
+sphinx_rtd_theme
+traitlets>=4.1
+recommonmark==0.4.0
+pyyaml
+sphinx-copybutton
+setuptools

--- a/readthedocs.yml
+++ b/readthedocs.yml
@@ -1,6 +1,6 @@
 name: kubespawner
 type: sphinx
-conda:
-  file: docs/environment.yml
+requirements_file: docs/requirements.txt
 python:
   version: 3
+  setup_py_install: true

